experimental/PropertyDDS/packages/property-dds/src/test/rebasing.spec.ts

```diff
@@ -130,7 +130,7 @@ describe("PropertyDDS", () => {
 					try {
 						const numOperations = random.irandom(maxOperations);
 						const maxCount = operationCumSums[operationCumSums.length - 1];
-						for (const j of _.range(numOperations)) {
+						for (const _j of _.range(numOperations)) {
```

Comment:
Weird loop -- Probably should get a closer look (e.g. why is this using _.range at all) but for a more narrow fix --

Looks like .range should return an array, so something like this would be a good option
```suggestion
						_.range(numOperations).forEach(() => {
```